# frame_type: 0x2A(42)
# power: 0 -> OFF, power:1 -> ON
int32 power